internal/cli/inputs.go

```diff
@@ -116,6 +116,41 @@ func ResolveApp(ui terminal.UI, client realm.Client, filter realm.AppFilter) (re
 	return appsByOption[selection], nil
 }
 
+// ResolveApps will use the provided Realm client to resolve multiple apps specified by the filter
+func ResolveApps(ui terminal.UI, client realm.Client, filter realm.AppFilter) ([]realm.App, error) {
```

Comment:
Thoughts on a separate function here? I felt that the different return types/slightly different survey made this fine to just duplicate and repurpose

Reply:
do we really use `ResolveApps` anywhere else besides `app delete` though?  this should probably just live in the `deleteInputs` as regular old code

and on your other question about `--app` being a string here, that's probably indication enough that we don't want to use `cli.ProjectInputs` either

Reply:
Refactored and added this onto deleteInputs